test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1

```diff
@@ -24,15 +24,21 @@ Describe "Can load a native assembly" -Tags "CI" {
         if ($IsWindows) {
             $arch = "win-" + $processArch
             $nativeDllName = "nativedll.dll"
-            $sourceDllName = "hostpolicy.dll"
+            $sourceDllName = Join-Path $PSHome "hostpolicy.dll"
```

Comment:
I guess this dll doesn't exist under `$PSHOME`, because it's part of the .NET runtime.

Reply:
We don't currently test a FX Dependent package on Windows, so it should be in PSHome